src/client/client4.js

```diff
@@ -1143,6 +1143,13 @@ export default class Client4 {
 
     // Channel Routes
 
+    getAllChannels = async () => {
```

Comment:
Should there be `page` and `perPage` parameters here?

Reply:
Yes, I think so. I going to review that.

Reply:
Fixed